core/modules/user/tests/user_password_reset.test

```diff
@@ -204,4 +207,87 @@ class UserPasswordResetTest extends BackdropWebTestCase {
     $this->assertNoText('You have tried to use a reset password link that has either been used or is no longer valid. Please request a new one using the form below.');
   }
 
+  /**
+   * Test user-based flood control on password reset.
+   */
+  public function testPasswordResetFloodControlPerUser() {
+    // Set a very low limit for testing.
+    config_set('user.flood', 'flood_user_limit', 2);
```

Comment:
I suggest using a variable for the limit and reuse it later on with the `for` loops. This is a cleaner and more manageable way.

```suggestion
    $limit = 2;
    config_set('user.flood', 'flood_user_limit', $limit);
```

Reply:
Good idea, readability is always important. :+1: